docs/rules/prefer-arrow-callback.md

```diff
@@ -1,77 +1,100 @@
-# Suggest using arrow functions as callbacks. (prefer-arrow-callback)
+# Suggest using ES6 arrow functions to describe callbacks (prefer-arrow-callback)
```

Comment:
We should also make sure this matches the metadata property in the rule itself!: https://github.com/eslint/eslint/blob/master/lib/rules/prefer-arrow-callback.js#L136